docs/INSTALL.md

```diff
@@ -74,28 +74,40 @@ command line scripts as explained in the [scripts README](https://github.com/Ada
     ```
     /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
     ```
-3) Install python3 and libsodium
+3) Install automake, libtool, and libsodium
     ```
-    brew install python libsodium
+    brew install automake libtool libsodium
     ```
-4) Create virtualenv "jmvenv"
-    ```sh
-    pip3 install virtualenv
-    virtualenv jmvenv
-    source jmvenv/bin/activate
+4) Build secp256k1
+    ```
+    git clone https://github.com/bitcoin-core/secp256k1
```

Comment:
For consistency with the Linux install, we should choose the same commit, see: https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/install.sh#L256

Btw as I explained in some other thread, I chose this commit only because it was the one used by coincurve; there is no release cycle for secp256k1.